website/docs/MigrationV12.md

```diff
@@ -0,0 +1,28 @@
+---
+id: migration-v12
+title: Migration to 12.0
+---
+
+Migration to React Native Testing Library version 12 from version 11.x should be a relatively easy task due small amount of breaking changes.
+
+# Breaking changes
+
+1. `*ByRole` queries now return only accessibility elements, either explicitly marked with `accessible` prop or implicit ones where this status is derrived from component type itself (e.g `Text`, `TextInput`, `Switch`).
+
+You may need to adjust relevant components under test to make sure they pass `isAccessibilityElement` check.
+
+2. `*ByText`, `*ByDisplayValue`, `*ByPlaceholderText` queries now return host components, as all other queries.
+
+While potentially breaking, this should not cause issues in tests.
```

Comment:
What if it does? Do we know how this can break and guide users on how to do it?

Reply:
Hontestly, we do not have a clear case when it breaks, as composite & host `Text` / `TextInputs` are very simillar in terms of props. For the types of test we recommend (query + jest matchers) it should work the same. The case where it might break is when user did e.g. explicit type checks, or inspects children, etc

Reply:
> The case where it might break is when user did e.g. explicit type checks, or inspects children, etc

To me it's something that's worth mentioning